### PR TITLE
[WDPT REMOTE 0602] Lynn W-B

### PR DESCRIPTION
**Pull request by lynn-w-b**
Iterations 1-4 complete (without styling)

### CODE REVIEW

**Comment by lynn-w-b** [comment]
Got as far as I could with the styling in the time I have available. Can you please give me some feedback re the getting the beer id from the url part? Maybe we could chat this through? I followed the instructions online and it works but I'm not sure I quite grasp why. Thanks

File: app.js [hunk 1]
```diff
@@ -11,15 +11,33 @@ app.set('view engine', 'hbs');
...
+  .catch(error => console.log(error));
+});
+app.get("/random-beers", (req, res, next) => { 
+   punkAPI.getRandom()
+   .then(randomFromApi => { 
+    res.render("random-beers",{randomFromApi});
+   })
+   .catch(error => console.log(error));
+ });
+ app.get("/getbeer/:id", (req, res, next) => {
```
Comment (valmgisbert) [inline app.js:33]: good structure in calling the ID!

File: views/random-beers.hbs [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<div id="random">
+    {{#each randomFromApi}}
+        {{> beerpartial this}}
+        <div id="food">
+        <h5>Food Pairing:<br></h5>
+        <ul>
+        {{#each food_pairing}}
+        <li>{{this}}</li>
```
Comment (valmgisbert) [inline views/random-beers.hbs:8]: work a bit on the spacing here. All is good tho!

**Comment by valmgisbert** [comment]
Great work Lynn! and sure! what you're doing in it 6 is making a request to the API with a certain parameter: in this case their id. By looking through their beer array, that request plucks the info of the one matching the id inside of the target, and sends it to be reflected in your hbs file. If you need any more information about this, please let me know!

**Pull request closed without merging**